### ISSUE TITLE
TreeTable: aria-level attribute not allowed

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#15006]
### Describe the bug

Aria-level attribute is supported with treegrid rows, but not table. Add missing roles to the treeTable html.
• The treegrid container has role treegrid.
• Each row container has role row and is either a DOM descendant of or owned by the treegrid element or an element with role rowgroup.
https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/

### Environment

https://primeng.org/treetable

### Angular version

17.0.0

### PrimeNG version

17.10.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Steps to reproduce the behavior

https://primeng.org/treetable

### Expected behavior

https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/
